app/views/layouts/tailwind.html.erb

```diff
@@ -41,6 +41,14 @@
     <%= stylesheet_pack_tag 'layouts/tailwind', media: 'all', 'data-turbolinks-track': 'reload' %>
     <%= stylesheet_pack_tag 'IconFirstPack', media: 'all' %>
 
+    <% if Rails.env.test? %>
+      <style>
+        * {
```

Comment:
```
* {
   animation: none !important;
   transition: none !important;
   }
```
@harigopal Can we try disabling the _animation_ and _transition_ properties in test env?

Reply:
@vinutv This won't work properly everywhere. An example of a failure this causes is the editor drawer, which animates _in_ from off-screen. If animations are disabled altogether, the drawer's body will be off-screen, and Capybara will only interact with visible elements by default (the safe option).

I didn't realize transitions were controlled separately, though. I guess I'll also need to add `transition-duration: 0s !important;` to the list. 🤔